pombola/south_africa/views/api.py

```diff
@@ -0,0 +1,21 @@
+from django.http import JsonResponse
+
+from pombola.core.models import Organisation, ContactKind
+
+
+# Output Popolo JSON suitable for WriteInPublic for any committees that have an
+# email address.
+def committees_popolo_json(request):
+    committees = Organisation.objects.filter(
+        slug__contains='committee',
+        contacts__kind__slug='email'
+    )
+    popolo = []
+    for committee in committees:
+        popolo.append({
+            'id': committee.id,
+            'name': committee.name,
+            'email': committee.contacts.filter(kind__slug='email')[0].value,
+            'contact_details': []
+        })
```

Comment:
I think patterns like this (create a list and append to it in a loop) are less error prone to write as list comprehensions